orbit/cmd/desktop/desktop.go

```diff
@@ -296,6 +297,8 @@ func main() {
 		defer tic.Stop()
 		go func() {
 			<-deviceEnabledChan
+			tic := time.NewTicker(5 * time.Minute)
+			defer tic.Stop()
```

Comment:
I'm confused by this `tic` here, which has the same name as the one in the parent scope. Could you help me understand its purpose?

Reply:
Oh good catch, this was added by mistake. I don't remember adding it, must have been an inadvertent Vim yank and paste! 🤦

Reply:
@roperzh I think the mistake I made in adding those two lines was masking an issue with the change made here: https://github.com/fleetdm/fleet/pull/21127/files

when I remove those two lines, fleet desktop never sends an initial call to the desktop summary endpoint. Once I click My device, it sends the call. Any ideas why? I can't seem to find a reason.